src/display.h

```diff
@@ -5,3 +5,5 @@ extern const char* version_msg;
 
 void cp_usage();
 void cp_version();
+
+char* cp_spacer(int len, int padding);
```

Comment:
It isn't immediately clear what this function does, and it's even more confusing because I don't see a function declaration for this. That means another file could use this but compilation would fail.

Reply:
Yea, I was making something that made equal space between the token type and contents, but I could not figure it out. I forgot to delete it from the header file.